pkg/flare/archive.go

```diff
@@ -621,6 +626,45 @@ func zipSecrets(tempDir, hostname string) error {
 	return writeScrubbedFile(f, b.Bytes())
 }
 
+func zipProcessChecks(tempDir, hostname string, getAddressPort func() (url string, err error)) error {
+	addressPort, err := getAddressPort()
+	if err != nil {
+		return fmt.Errorf("wrong configuration to connect to process-agent")
```

Comment:
You may want to preserve the `err` in the message here or do `errors.Wrap`

Reply:
Good point, I will do that